test/libcommonserver/io/testcheckifpathexists.cpp

```diff
@@ -96,10 +96,45 @@ void TestIo::testCheckIfPathExistsSimpleCases() {
         CPPUNIT_ASSERT(exists);
         CPPUNIT_ASSERT(ioError == IoErrorSuccess);
     }
+
+    // A regular file withour read/write permission
+    {
+        LocalTemporaryDirectory temporaryDirectory("TestIo");
+        const SyncPath path = temporaryDirectory.path() / "test.txt";
+        std::ofstream ofs(path);
+        ofs << "Some content.\n";
```

Comment:
Is it useful?

Reply:
[4f58023](https://github.com/Infomaniak/desktop-kDrive/pull/239/commits/4f58023ff3bd5aff09ada2606e377a3d08b28a9b)